src/Core/Tool/Authorizer/SetAuthorizer.php

```diff
@@ -92,7 +92,7 @@ public function isAllowed(Entity $entity, $privilege)
 		}
 
 		// Finally, all users can search sets
-		if ($privilege === 'search') {
+		if ($user->getId() and $privilege === 'search') {
```

Comment:
not logged in users should be able to see lists of collections (sets), but only those that are public

Reply:
the same applies to saved searches endpoint. That seems to be working OK in develop (filtered at another stage? )